ui/e2e/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Need a require for eslint config.
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const baseConfig = require('../.eslintrc.base.js');
+
+module.exports = {
+  ...baseConfig,
+  extends: [...baseConfig.extends, 'plugin:playwright/playwright-test'],
```

Comment:
This plugin is really helpful for catching when you need `await`s and some other bits like that, which are easy to miss and lead to confusion with test behavior.